lib/matplotlib/tests/test_usetex.py

```diff
@@ -3,13 +3,20 @@
 
 import matplotlib
 from matplotlib.testing.decorators import image_comparison
+from matplotlib.compat.subprocess import check_output
 import matplotlib.pyplot as plt
 
 
 @image_comparison(baseline_images=['test_usetex'],
                   extensions=['pdf', 'png'],
                   tol=0.3)
 def test_usetex():
+    cmd = ['latex', '-version']
+    try:
+        check_output(cmd)
+    except:
+        from nose import SkipTest
+        raise SkipTest('missing command: %s' % cmd[0])
```

Comment:
Why not to use [`skip_if_command_unavailable`](https://github.com/matplotlib/matplotlib/blob/master/lib/matplotlib/testing/decorators.py#L441) decorator?